profiles/common/modules/features/communities/communities.install

```diff
@@ -18,11 +18,14 @@ function communities_disable() {
  */
 function communities_enable() {
   // SolR configuration add bundle.
-  db_insert('apachesolr_index_bundles')->fields(array(
-    'env_id' => 'solr',
-    'entity_type' => 'node',
-    'bundle' => 'community',
-  ))->execute();
+  if (module_exists('apachesolr')) {
+    db_insert('apachesolr_index_bundles')->fields(array(
+      'env_id' => 'solr',
+      'entity_type' => 'node',
+      'bundle' => 'community',
+    ))->execute();
+  }
+
```

Comment:
you should also remove the dependencies from community profile after watching this change, shouldn't you?